azure-pipelines.yml

```diff
@@ -12,44 +12,51 @@ pr:
+HgGD9L/LBiGtwEFAh9Bt109y+6WLQRvak8nkPNYOJGP11/bpoU/ZM/vYtnIPgjlVrHBpl81fwOoAcHydaCx8MEEpsfwTD+hlLveVmYi9h2B+ubyjLUN+B8uC8H6/xyw8/DbUtomRHXi3aVXFEon+b6wfUZjEX/xT7q2ORViFW40WNWnRWsOO2NQVc4Q9ZhN9LsNSModi3ULVorXWTo69hCOprDPOEv1WYy2Dh1FjFhpH2q/i8i7n3UP1IIkufJ3fsAzzZnMm2TGcH5J06ty7DJnBxofaVZXJk5GFhmHv89xRL56XswrmYS/VQQkoOH2IpLbmx9uV6EwLrwllocsA154hathQjk8rlUvCGi/scNeKyz/msmff76JXqHfVBFHOXgybORc3a5Z2iZjDac7q1pC5GY36pAvBrv97TWsivfqN/DSeprajAqmfc4hJmaH7pA4lgzDorV644woTv4VMjcmEFmWXAnbL1MSPEZKLSmnnUvdwy1/9vhITX3kaLmAn6u9Fv6aSZqsYb2JkEYmJ4ff9nc4MtqDg6mAsu1NJJEEH3jiizOWTCrjbEQTLn8aU2oDN/+xmTLAuFbHl5roVbamckUgUKFlAubh8Tc78WpamYzRJysVFjqvyYVAasgkJkVcNkV8JMmmp0Q3uZGyg0vAlFN/iEEGgeMT+clfYhfHh4joW+ro0+4Wr3rolvwMc5XElH/APEhta1pRVcvkVQlXpIQbPalbNJMFrnqkwBOohKeZDpf23ec5CZDSCYBskiMkNAoqiyIHmHVA9GVtUsnOaYL2L4b6abrOIJ//awPhGQJeW6YzL21g06kare+yhRM6el9lxAVWaonoegMPp83r4qRg2la9GMQRkHCJ1zXQ1XMSgdIIIb/bSdDu9QVQCdLCbuV5lb8K/7IXaCSVeAS24c7uJum/HE3USCU9HJha+lemDSxPS3I+JipvCZR45EduGtXO+gffOGsAz+pRdIWe1WKouExOKBrYNkKYJU7evF/0IQIa4BCHW7+aZpqmRjT61hFMUcXkNxLM7SXyBDWw7u7xnh5zrmOhyUS4bm0cHCtqaD/E1gYiAfy0mVtkWkRapn/EbqLIGZlg8cRFbMv6WssAw+9DRR84qgae1DHjxEAzwbDIZVyhbGQqDteVv/pmZwWBZ3+XGQEmlLV9ecLbkRIFVEgSQDL2FSNuAdp6rDCyyNFelNLyEPUcpsG5b5NjSP0YtKkECmWHaXUIDNtqiqFSZVmSoD9/PVkJkBPO0BB0G+pn3UNZd1zX0kYOp59bAtBKSTyGdJ52rdg9TsoNh9btK3vDhvIA16k9ZuhWfDC01KJpuKYmuxcKJ4O5n2gQuuq9lzqBO6hR44ST8LcAnDt2pp3O7q18/4LkoxJUxWzW1dmNn4CNVemwHShwk4m5g4XdjzVR/A4kRAwVZ2RsMpq5ZyRtbmacdquFGfOEfamRJEAiJ//E0TJsAJJfpzMQoTM/AgpFncS3SHsMVia+dPIXuRUrag+00POCzBlDVn91yGIWFhm01BB8+ahgsPfl7k4QInXBYHAW4drXZvFZ9F90O6o2Hcym3YRVD3Mp3UWacGNDHVGviiMAyU3aDrOYGZlHjNVvlL/+zc0YWLiwOdAZesc/sWP0PDMY1NUXHQi7ln/gOVZTbYdybxgnspXZYsTSaHloMB3Wd78doyzIjR1LdcbD+UBC4cSxAvs1AcozZ62Vv2muKit3ZtnktLCLzrCJ6nENkFsm7cnit64LuRn3HgCMSt/+HcRB3tgZFu6hyzHcf47uLI1oylRb8N0/l/DYMDWAezSmHXdX17YULePcrWNSkcULzmXw5xRWbLPd7M7IHLfKYbuCB06PZ3rmLyM13LbQrzsMEeNDuaTu4aGSP95CJiIiAVt9jZ6ZSWw3FHm8mTvs0WE2JXRC3SRzLuk5vSAtMz2iKQ4EscejDGcyyKFsMkpmBdIsT53UEP9CCfqywht5G7v6AgHrWPeZIYqiZc2+j4JhfjkcqxdreEXaL6lIl1fvZq1dq+nLWNgVrIh/BsWxjhkIS2zuoqx8PQra2rXsZX2iDQj6hQ/1aOfuUqvXHabfST8+xgdqP0U8sP29Z4mlXo+xhSN9yIb+r9bp2QMmGbgzxpU86SnSMiJwoGCp/r9U10jLVxIjtL1NqlpL3ouCFYC6FXBWi8MW2ZDEYJ3gB5WlSVHXBX5+o5Wu4LE947YVwRWDS2sdt26tUYw2GHiO0SlUYvc3bSCtK9poDuFY0f7ufsaz8QyXDooADxIM40e3PaCSg/YvItBMaOFaVHHzaGNEt+pmrcohvJVIgXF7B4SELcCnC2Moye8yGpGy+ODnocFcuGK6rkjudElIFIfbI/9NKCtzcen41b+69D/CpxX3yaEZPw1Tb6+80X57Zozy0WIGfv6mTADD7GS3dG5pcW59y5cG/aVZcTBlWyB2TECyBf5Bm8iHyeSMdUkKjHSisoRdP0OWXXQI9Xxvy40QkmmcagN5fEcwVnZjsRcMDB2p9FqiX6aGOXdEeIvIgNLHoDjw1HCMqVO4UAuZ74dyLmHAn+Jc5393DWpH5sSQbcMuZ/3O9FqCRt94qG1Wbnp/DUTsy71OiH11M9K9HSGwkVbwiz1oehMt42/IBhJ/2WrJ6lldflD7o+3RtroDcR6cb8oBYqSjoMJOUdxzzLzwqBrQynLKAIvRFEE4CA9A7auRzy9nu+EyVdc2E+OwIB8ymVa16K/OjRK+hIClF80t5O41iUz6019hyLb39Bk8T0oEPv8naNnUhb2gqPBcHWastSXnpdLA4jeUwbp60X0jgw9k+aQuYJHeX8Tix0EMIp0wcBDC7Sp7CEmd2Rvpovtx1w51yadDZ1IsWgxAaCByGz6KibwwOpxPHAfPo0P9BBtdBqPTYAIc6t280CLMD4wITAJBgUrDgMCGgUABBQTVTI0xnCLmO1UYS1iD9ZPVhaeuQQU0b9I3cUSd9/VfpENPWDQ4kn8FosCAwGGoA=='
```

Comment:
Why is this needed?

Reply:
It's good practice to rely on a trust store when hostname validation is turned off as it must be to access multi-region Cassandra API instances. Driver-3 turns hostname validation off by default. Driver-4 turns hostname validation on by default. We choose to use turn hostname validation off and use a trust store.